README.md

```diff
@@ -35,7 +35,20 @@ If you also need frontend for development, check out [frontend repository](https
 
 ## Tests
 
-Tests can be run with tox automation: just run `tox` on project root (remember to install it first with `pip install tox`).
+Tests can be run with tox automation: just run `tox -p auto` on project root (remember to install it first with `pip install tox`).
+
+## Developing
+
+### Developing with VS code
+
+VS code provides functionality to develop inside docker container. This mitigates the need to install development environment and dificulties to make things work with different OSs. Also developing inside container gives you ability to see code changes on the fly. 
+
+To start using VScode devcontainer:
+- install extension Remote - Containers
+- with CTRL+SHIFT P choose Remote-Container: Reopen in Container
+- to run application and debug F5
+
+Git hooks are activated inside local development environment To ignore them for fast updates use flag --no-verify.
```

Comment:
for using docker with `buildx` i  needed to:

```
wget https://github.com/docker/buildx/releases/download/v0.7.0/buildx-v0.7.0.linux-amd64
mkdir -p ~/.docker/cli-plugins
cp ~/Downloads/buildx-v0.7.0.linux-amd64 ~/.docker/cli-plugins/docker-buildx
chmod +x ~/.docker/cli-plugins/docker-buildx
```

also to add to `~/.docker/config.json`
```
"experimental": "enabled",
```

after that it worked

Reply:
will add this to readme